lib/puppet/face/node/clean.rb

```diff
@@ -52,7 +52,7 @@ def debug(message)
     end
 
     def warn(message)
-      Puppet.warning(message)
+      Puppet.warning(message) unless message =~ /cadir is currently configured to be inside/
```

Comment:
@puppetlabs/puppetserver-maintainers , there was a conflict here, do we need to keep the `unless`?

Reply:
yes